EasyIntern-Backend/Controllers/AccountController.cs

```diff
@@ -150,7 +150,7 @@ public async Task<IActionResult> Login([FromBody] JsonModelAccountLogin model)
             ModelState.AddModelError("AuthenticationError", safeReturnError);
         if (!ModelState.IsValid)
         {
-            return Json(ModelState);
+            return BadRequest(ModelState);
```

Comment:
This should return 401 unauthorized instead of 400 bad request (or 418 im a teacup)